src/RoslynPad.Editor.Windows/Shared/RoslynCompletionData.cs

```diff
@@ -35,6 +35,12 @@ public async void Complete(TextArea textArea, ISegment completionSegment, EventA
             return;
         }
 
+        var textArgs = e as CommonTextEventArgs;
+        if (textArgs != null && textArgs.Text?.Length > 0)
+        {
+            return;
+        }
+        
```

Comment:
This is incorrect, since completion can be triggered without typing. Please remove